app/views/api/applications/edit.html.slim

```diff
@@ -12,6 +12,6 @@ h2
     = form.user_defined_form
 
   = form.buttons do
-    = form.commit_button
+    = form.button 'Update Application', button_html: {class: 'pf-c-button pf-m-primary' }
```

Comment:
Same as before:
I may be wrong but I think it has to be either commit_button or have type: 'submit'.